runtime-examples/background.js

```diff
@@ -0,0 +1,21 @@
+let loadTime = new Date();
+let manifest = browser.runtime.getManifest();
+
+function onInstalledNotification(details) {
+  browser.notifications.create('onInstalled', {
+    title: `Runtime Examples version: ${manifest.version}`,
+    message: 'onInstalled has been called',
+    type: 'basic'
+  });
+}
+
+function onClick() {
+  browser.runtime.reload();
+}
+
+browser.browserAction.setBadgeText({
+  text: `${loadTime.getMinutes()}:${loadTime.getSeconds()}`
```

Comment:
This gets truncated at 4 characters, so it's confusing (It shows something like "47.2"). I don't think you really need this loadTime/badge text stuff, do you? I'd be tempted to omit it.

Reply:
Really? For me it shows all 5 characters. I wonder why.

We don't really, I just wanted to show the effect of reload in some way. I could add it to the notification message instead where it won't be truncated.

Reply:
That is weird.

>  I just wanted to show the effect of reload in some way.

I think seeing the notification again works for that, doesn't it?

Reply:
It shows that the install notification has been called, but not that the background page (and other stuff) has been reloaded as well.